src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -961,6 +961,65 @@ public override SyntaxTree SyntaxTree
 
         internal override IOperation GetOperationWorker(CSharpSyntaxNode node, GetOperationOptions options, CancellationToken cancellationToken)
         {
+            var bindingRoot = GetBindingRootOrInitializer(node);
+
+            var statementOrRootOperation = GetStatementOrRootOperation(bindingRoot, options, cancellationToken);
+            if (statementOrRootOperation == null)
+            {
+                return null;
+            }
+
+            // we might optimize it later
+            return statementOrRootOperation.DescendantsAndSelf().FirstOrDefault(o => !o.IsImplicit && o.Syntax == node);
+        }
+
+        private CSharpSyntaxNode GetBindingRootOrInitializer(CSharpSyntaxNode node)
+        {
+            var bindingRoot = GetBindingRoot(node);
+
+            // if binding root is parameter, make it equal value
+            var parameter = bindingRoot as ParameterSyntax;
+            if (parameter != null && parameter.Default?.FullSpan.Contains(node.Span) == true)
+            {
+                // we need to do this since node map doesn't contain bound node for parameter
```

Comment:
probably move this comment outside the if and just after the previous comment?

Reply:
okay.